[EDU-3821] [EDU-3809] refactor: adjust links (#659)
aziontech/docs: src/content/docs/en/pages/changelog/previous-years/previous-years-changelog.mdx

**Data Streaming**

> * **IBM QRadar**: a new  connector available for integration. Check it out [here](/en/documentation/products/observe/data-streaming//#ibm-qradar) and learn how to configure your IBM QRadar endpoint to receive data collected via Data Streaming.

Reviewer: ```suggestion
* **IBM QRadar**: a new  connector available for integration. Check it out [here](/en/documentation/products/observe/data-streaming/#ibm-qradar) and learn how to configure your IBM QRadar endpoint to receive data collected via Data Streaming.
```

**Data Streaming**

> * **Data Dog**: a new  connector available for integration. Check it out [here](/en/documentation/products/observe/data-streaming//#datadog) and learn how to configure your Datadog endpoint to receive data collected via Data Streaming.

Reviewer: ```suggestion
* **Data Dog**: a new  connector available for integration. Check it out [here](/en/documentation/products/observe/data-streaming/#datadog) and learn how to configure your Datadog endpoint to receive data collected via Data Streaming.
```

**Edge Application**

> * A new feature, [Slice configuration](/en/documentation/products/build/edge-application/edge-caching/#slice-settings), is available on Edge Application.

Reviewer: ```suggestion
* A new feature, [Slice configuration](/en/documentation/products/build/edge-application/edge-caching/#file-slicing), is available on Edge Application.
```

* Added "host" variable to Real-Time Events for Data Source "Edge Application".
* Bug fix in the Real-Time Events interface.
* Added the graph referring to the "Data Streamed" metric of the Data Streaming product, in the Real-Time Metrics interface.
* New Bot Management solution integrated to the Azion platform, using [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/) in [Edge Firewall](/en/documentation/products/secure/edge-firewall/).

> * Added [TLS Minimum Version](/en/documentation/products/build/edge-application/#minimum-tls-version) selection component to the Edge Application interface.

Reviewer: o # não funciona

> * Added [TLS Minimum Version](/en/documentation/products/build/edge-application/#minimum-tls-version) selection component to the Edge Application interface.

Reviewer: ```suggestion
* Added [TLS Minimum Version](/en/documentation/products/build/edge-application/#digital-certificates) selection component to the Edge Application interface.
```
